File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/go/josevalver.go
package main

import (
	"fmt"
)

func main() {
	for i := 1; i <= 100; i++ {
		if i%3 == 0 {
			fmt.Println("fizz")
		} else if i%5 == 0 {
			fmt.Println("buzz")
		} else if i%3 == 0 && i%5 == 0 {
			fmt.Println("fizzbuzz")
		} else {
			fmt.Println(i)
		}
	}
}

File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/rust/josevalver.rs
fn main() {
    for i in 1..101 {
        if i % 3 == 0 {
            println!("fizz");
        } else if i % 5 == 0 {
            println!("buzz");
        } else if i % 3 == 0 && i % 5 == 0 {
            println!("fizzbuzz");
        } else {
            println!("{}", i);
        }
    }
}